docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3"
+services:
+
+  site:
+    image: dart-dev:stable
+    build:
+      context: .
+      target: dev
+    command: ["make", "serve"]
+    env_file: .env
+    environment:
+      - JEKYLL_ENV=development
+      - JEKYLL_SITE_HOST=0.0.0.0
+      - JEKYLL_SITE_PORT=4000
+      - NODE_ENV=development
+    ports:
+      - "4000:4000"
+      - "35729:35729"
+      - "5000:5000"
```

Comment:
What is this 5000 port used for? It seems AirPlay on macOS uses it by default, preventing me from running the site. I can disable "AirPlay receiver" temporarily or modify this file allowing the site to run, but I'd imagine enough users will hit this issue.

Reply:
Oh you already committed a fix for this, thanks!

Reply:
Yup, all set. I moved all Firebase emulation to 5500 to avoid this. Good catch.